lib/coinbase/staking_operation.rb

```diff
@@ -4,18 +4,37 @@ module Coinbase
   # A representation of a staking operation (stake, unstake, claim rewards, etc). It
   # may have multiple steps with some being transactions to sign, and others to wait.
   # @attr_reader [Array<Coinbase::Transaction>] transactions The list of current
+  # @attr_reader [Symbol] status The status of the operation
+  # @attr_reader [String] error The error message if the operation failed
   #   transactions associated with the operation
   class StakingOperation
-    attr_reader :transactions
+    attr_reader :transactions, :status, :error
+
+    # Returns all StakingOperations.
```

Comment:
hmm I don't think we currently support this in the backend ... only support a Get StakingOperation currently

Reply:
@drohit-cb should this be removed?

Reply:
I will confess - my ruby knowledge is bad - was trying to go through the ruby sdk code to see if this had any purpose when Dean added it ?

because the backend does not support listing staking operations as of today...

If you think so as well - I am happy to remove it 👍

Reply:
have remove this code now.